src/basic/gunicode.c

```diff
@@ -6,6 +6,7 @@
  */
 
 #include "gunicode.h"
+#include "macro-fundamental.h"
```

Comment:
Hmm, this file is extrinsic, hence I doubt our usual coding style applies here. But at the very minimum we prefer `macro.h` instead of the -fundamental version here.